### DISCUSSION TITLE
大家都是用什么工具和机场来解锁的，最新版能正常使用吗？

### DISCUSSION THREAD

**Discussion by Jade-Doraemon** [Semporia/TikTok-Unlock#243] [Q&A] [answered]
我现在用的 Quantumult X 搭配 Dler Cloud , 目前 TikTok 25.4 一切正常。

**Comment by Semporia** [comment, marked as answer]
Quantumult X 、 Surge 、 Loon 、Shadowrocket 测试均可解锁最新版25.4，直播功能会受部分机场节点影响，请尝试更换节点或机场！

**Reply by seoho8908** [reply]
我目前使用的surge，最新版Tiktok正常使用。
不过GitHub有很多人反馈圈X，小火箭等解锁失败。不知道什么原因。

**Reply by Semporia** [reply]
Quantumult X TF 1.0.31（741）
Loon TF 2.2.0（440）
Shadowrocket 2.2.11
这几个版本测试都正常使用。

**Reply by aliwanggg** [reply]
Quantumult X 1.0.30-707按照此方式安装后，无法使用，提升无网络连接。
iPhone6S，IOS12.5.7 还是在无插卡的情况下配置quanX和安装tiktok都无法使用。求问大佬知不知道问题出哪里了。
TikTok 21.1.0和更低的TikTok 20.1.0都试了不行。

**Reply by Semporia** [reply]
> Quantumult X 1.0.30-707按照此方式安装后，无法使用，提升无网络连接。
> 
> iPhone6S，IOS12.5.7 还是在无插卡的情况下配置quanX和安装tiktok都无法使用。求问大佬知不知道问题出哪里了。
> 
> TikTok 21.1.0和更低的TikTok 20.1.0都试了不行。

無卡情況下，直接使用分流即可。

無網絡的日誌貼一下。

**Comment by Kaiser-DMr** [comment]
机场没关系，刚刚用surge 解锁 +升级没问题

**Comment by PublicMing** [comment]
目前使用的shadowrocket，抓包的21.0版本可以正常使用，一旦更新版本后就无法刷出视频，提示无网络连接

**Reply by Semporia** [reply]
Quantumult X 、 Surge 、Loon 、 Shadowrocket 
最新版本功能都正常使用

**Reply by nivranic** [reply]
更新后第一次打开还能连上，中途被杀后台就重新打开就无网络了，这难道是一次性的？

**Reply by leehao321** [reply]
我也是这样，不能升级一升级就要重新弄> 更新后第一次打开还能连上，中途被杀后台就重新打开就无网络了，这难道是一次性的？

**Reply by wanfdefuo** [reply]
+1 一换网就不行了 无网络连接 换回来也不行 怎么弄都不行
